### PR TITLE
874348 - UG documentation for Serialization in WPF Image Editor

### PR DESCRIPTION
**Pull request by Shalini-Ashokan**
### Description
UG documentation for Serialization in WPF Image Editor.
Content review task : https://dev.azure.com/Syncfusion-General/Documentation/_workitems/edit/765187

### CODE REVIEW

File: wpf/Image-Editor/serialization.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: Serialization in syncfusion SfImageEditor WPF.
```
Comment (karthikraja-arumugam) [inline wpf/Image-Editor/serialization.md:3]: This section describes how to serialize and deserialize the annotations in SfImageEditor control for the WPF platform.

File: wpf/Image-Editor/serialization.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+description: Serialization support in Syncfusion Essential Studio WPF ImageEditor (SfImageEditor) control, its elements and more.
+platform: wpf
+control: SfImageEditor
+documentation: ug
+---
+
+# Serialization in SfImageEditor
```
Comment (karthikraja-arumugam) [inline wpf/Image-Editor/serialization.md:10]: # Serialization in Image Editor (SfImageEditor)

File: wpf/Image-Editor/serialization.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+The Image Editor control provides support for serializing and deserializing annotations, freehand drawing, text, and toolbar settings. You can save the current state of the image editor and load it back when needed.
```
Comment (karthikraja-arumugam) [inline wpf/Image-Editor/serialization.md:12]: The Image Editor control provides support to serialize and deserialize the shape, text, pen annotations, and custom view along with their settings. You can save the current state of the image editor annotations and load it back when it is needed.

File: wpf/Image-Editor/serialization.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+## Serialization
+
+The `Serialize()` method is used to serialize the current edits of shapes. It allows you to store the `SfImageEditor` annotation settings using the `Serialize` method by passing the stream as a parameter.
```
Comment (karthikraja-arumugam) [inline wpf/Image-Editor/serialization.md:16]: The Serialize method is used to serialize the current edits of annotations. It allows you to store the SfImageEditor annotations to the stream by passing the stream as a parameter to the Serialize method.

File: wpf/Image-Editor/serialization.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+{% tabs %}
+
+{% highlight C# %}
+    
+	//To Save as stream in file
```
Comment (karthikraja-arumugam) [inline wpf/Image-Editor/serialization.md:22]: remove comment

File: wpf/Image-Editor/serialization.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+    }
+    
+	
+{% endhighlight %}
+
+{% endtabs %}
+
+## Deserialization
+
+The `Deserialize()` method is used to deserialize the edits made on an image. It allows you to reload the `SfImageEditor` control with the settings available in the stream.
```
Comment (karthikraja-arumugam) [inline wpf/Image-Editor/serialization.md:42]: The Deserialize method is used to deserialize the annotations over an image. It allows you to reload the SfImageEditor control with the annotations available in the stream.

File: wpf/Image-Editor/serialization.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+    OpenFileDialog dialog = new OpenFileDialog();
+    if (dialog.ShowDialog() == true)
+    {
+        using (Stream myStream = dialog.OpenFile())
+        {
+            imageEditor.Deserialize(myStream);
+        }
+    }
+
+{% endhighlight %}
```
Comment (karthikraja-arumugam) [inline wpf/Image-Editor/serialization.md:58]: add topic

Annotations collection

Mention about read only collection

Comment (Shalini-Ashokan) [reply wpf/Image-Editor/serialization.md:56]: Added topic and mentioned read only collection

File: wpf/Image-Editor/serialization.md (hunk 6 above)
Comment (Shalini-Ashokan) [reply wpf/Image-Editor/serialization.md]: Updated

File: wpf/Image-Editor/serialization.md (hunk 5 above)
Comment (Shalini-Ashokan) [reply wpf/Image-Editor/serialization.md]: removed

File: wpf/Image-Editor/serialization.md (hunk 4 above)
Comment (Shalini-Ashokan) [reply wpf/Image-Editor/serialization.md]: Updated

File: wpf/Image-Editor/serialization.md (hunk 3 above)
Comment (Shalini-Ashokan) [reply wpf/Image-Editor/serialization.md]: Updated

File: wpf/Image-Editor/serialization.md (hunk 1 above)
Comment (Shalini-Ashokan) [reply wpf/Image-Editor/serialization.md:3]: Updated

File: wpf/Image-Editor/serialization.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+## Serialization
+
+The `Serialize` method is used to serialize the current edits of annotations. It allows you to store the `SfImageEditor` annotations to the stream by passing the stream as a parameter to the `Serialize` method.
+
+{% tabs %}
+
+{% highlight C# %}
+    
+    SaveFileDialog dialog = new SaveFileDialog();
+    dialog.Title = "Save XAML";
```
Comment (karthikraja-arumugam) [inline wpf/Image-Editor/serialization.md:23]: Save XML

File: wpf/Image-Editor/serialization.md [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+    dialog.Filter = "XAML File (*.xaml)|*.xaml";
```
Comment (karthikraja-arumugam) [inline wpf/Image-Editor/serialization.md:24]: dialog.Filter = "XAML File (*.xaml)|*.xaml";

use xml file

File: wpf/Image-Editor/serialization.md [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+    if (dialog.ShowDialog() == true)
+    {
+        using (Stream stream = File.Open(dialog.FileName, FileMode.CreateNew))
+        {
+            imageEditor.Serialize(stream);
+            imageEditor.Reset();
```
Comment (karthikraja-arumugam) [inline wpf/Image-Editor/serialization.md:30]: remove reset

File: wpf/Image-Editor/serialization.md [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+        }
+    }
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Annotations collection
+
+The Image Editor provides the read-only collection of annotations using the `Annotations` property.
```
Comment (karthikraja-arumugam) [inline wpf/Image-Editor/serialization.md:62]: The Image Editor provides the read-only collection of annotations using the `Annotations` property. The Annotations property contains all the annotations currently visible in image editor.

Note: This collection will be reset if the background image has been changed.

File: wpf/Image-Editor/serialization.md (hunk 9 above)
Comment (Shalini-Ashokan) [reply wpf/Image-Editor/serialization.md]: used xml file

File: wpf/Image-Editor/serialization.md (hunk 10 above)
Comment (Shalini-Ashokan) [reply wpf/Image-Editor/serialization.md]: removed

File: wpf/Image-Editor/serialization.md (hunk 11 above)
Comment (Shalini-Ashokan) [reply wpf/Image-Editor/serialization.md]: Updated

File: wpf/Image-Editor/serialization.md [hunk 12]
```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: Serialization in syncfusion SfImageEditor WPF.
```
Comment (florenceliliana) [inline wpf/Image-Editor/serialization.md:3]: title: Serialization in Syncfusion SfImageEditor WPF

**Review by florenceliliana — CHANGES_REQUESTED** [review]
The content review is completed.

File: wpf/Image-Editor/serialization.md (hunk 12 above)
Comment (Shalini-Ashokan) [reply wpf/Image-Editor/serialization.md]: Updated

File: wpf/Image-Editor/serialization.md [hunk 13]
```diff
@@ -0,0 +1,79 @@
...
+
+{% endtabs %}
+
+## Annotations collection
+
+The Image Editor provides the read-only collection of annotations using the Annotations property. The Annotations property contains all the annotations currently visible in image editor.
+
+N> This collection will be reset if the background image has been changed.
+
+The following code snippet only adds a rectangle shape from the `Annotations` collection.
```
Comment (florenceliliana) [inline wpf/Image-Editor/serialization.md:65]: The following code sample only adds a rectangle shape from the `Annotations` collection.

**Review by florenceliliana — COMMENTED** [review]
The content review is completed.

File: wpf/Image-Editor/serialization.md (hunk 13 above)
Comment (Shalini-Ashokan) [reply wpf/Image-Editor/serialization.md]: Updated

**Review by karthikraja-arumugam — APPROVED** [review]
This is fine

**Review by MuniappanSubramanian — APPROVED** [review]

**Review by Nijamudeen-Mohamed — APPROVED** [review]

**Pull request merged**